data/publiccloud/azure_more_cli/azure_vn.sh

```diff
@@ -107,8 +106,7 @@ do
         --vnet-name "${vnet}" \
         --output table
     echo "Done creating subnet ${subnet_names[$i]} with prefix ${prefix} and ${i}"
-    # shellcheck disable=2219
-    let "i+=1"
+    (( i++ )) || true
```

Comment:
```suggestion
    (( i++ ))
```

Why is `|| true` needed?

Reply:
fixed